documentation/src/main/asciidoc/topics/config_examples/jgroups_asym_encrypt.xml

```diff
@@ -0,0 +1,22 @@
+<infinispan>
+    <jgroups>
+         <stack name="encrypt-tcp" extends="tcp"> <1>
+           <SSL_KEY_EXCHANGE keystore_name="mykeystore.jks" <2>
+                             keystore_password="changeit" <3>
+                             stack.combine="INSERT_AFTER"
+                             stack.position="VERIFY_SUSPECT"/> <4>
+           <ASYM_ENCRYPT asym_keylength="512" <5>
```

Comment:
what kind of security are we trying to achieve here? RSA 512 bits isn't secure since 2000 and can be crack in a couple of hours currently.

Reply:
@pruivo Well spotted. Do you think we should change to 2048 or is there something better?

Reply:
2048 is fine. You have 4096 if you want to be safe for the next +1000 years or so xD